Sources/SwiftSgml/TagBuilder.swift

```diff
@@ -5,9 +5,14 @@
 //  Created by Tibor Bodecs on 2021. 07. 19..
 //
 
+
 @resultBuilder
 public enum TagBuilder {
     
+    public static func buildBlock() -> Tag {
```

Comment:
Sometimes a TagBuilder block contains optionals that all evaluate to nil. So a Tag is never returned. This allows for empty blocks.